### ISSUE TITLE
[Bug]: Quest Lags Client when loading quests

### ISSUE THREAD

**Issue by manmaed** [FTBTeam/FTB-Mods-Issues#1385]
### Mod

Quests

### Mod version

2101.1.1

### NeoForge / Forge / Fabric version

21.1.74

### Modpack & version

Evo 1.1.1

### What issue are you having?

When opening quests for the first time it lags the client out

### Crashlogs

N/A

### Steps to reproduce

Load a Quests

### Anything else to note?

Could be just because there's loads of quests/assets? mikey said to report it

**Comment by desht** [comment]
Yeah, it's a known issue, and probably not much can be done about it. It happens on big quest books on the first load after starting the client. I'll have a look and see if the asset loading can be smoothed out a bit in any case.

**Comment by manmaed** [comment]
Please disregard the previous reply from the stale bot, it's been a bad bot and sent for recycling.
